modules/renter/worker.go

```diff
@@ -240,25 +359,22 @@ func (w *worker) threadedWorkLoop() {
 
 // updateWorkerPool will grab the set of contracts from the contractor and
 // update the worker pool to match.
-func (r *Renter) updateWorkerPool(contracts []modules.RenterContract) {
-	// Get a map of all the contracts in the contractor.
-	newContracts := make(map[types.FileContractID]modules.RenterContract)
-	for _, nc := range contracts {
-		newContracts[nc.ID] = nc
-	}
+func (r *Renter) managedUpdateWorkerPool() {
+	_, currentContracts := r.hostContractor.ContractLookups()
```

Comment:
these receiver variables look backwards. Isn't `currentContracts` the first return value?